.github/workflows/manifold.yml

```diff
@@ -142,6 +142,24 @@ jobs:
         retention-days: 90
         overwrite: true
 
+  build_fuzzer:
+    name: fuzzer
+    timeout-minutes: 30
+    runs-on: ubuntu-24.04
```

Comment:
Can we try it on macos too?

Reply:
I think it will fail there due to the use of linux specific API (`gettid`). Need to look into what is the posix equivalent of it.

Reply:
Would this work? https://en.cppreference.com/w/cpp/thread/get_id

Reply:
I just forgot to change polygon fuzz. Should be working now.

I haven't yet checked if the thread can really be terminated, but so far we are not encountering cases where the code is inside an endless loop, so I guess we can check it later when needed...